engine/src/flutter/impeller/toolkit/android/surface_transaction.h

```diff
@@ -18,6 +18,22 @@ namespace impeller::android {
 class SurfaceControl;
 class HardwareBuffer;
 
+/// @brief A wrapper class that indicates whether a SurfaceTransaction was
+/// created by the flutter engine or was borrowed from Java for platform
+/// interop.
+struct WrappedSurfaceTransaction {
+  ASurfaceTransaction* tx;
+  bool owned;
```

Comment:
= false;

Reply:
true is the safer default here but roger.